File: codeWars/sumOfIntegers.js
function sumOfIntegersInString(s){
    let r = /\d+/g;
    let sum = 0;
  
    result = s.match(r);

    if (result == null) {
        sum = 0;
    } else if (result != null) {
        for(let i = 0; i <= result.length - 1; i++) {
            sum = sum + Number(result[i]);
        }
    }
    return sum
  }
  
  str = "The30quick20brown10f0x1203jumps914ov3r1349the102l4zy dog"
  
  console.log(sumOfIntegersInString(str));

File: codeWars/sum_of_integers.py
import re

str = "The30quick20brown10f0x1203jumps914ov3r1349the102l4zy dog"

def sum_of_integers(string):
    sum =  0
    numbers = re.findall(r'\d+', str)

    for i in numbers:
        sum = sum + int(i)
    
    return sum

print(sum_of_integers(str))
